### ISSUE TITLE
Problem with removing window

### ISSUE THREAD

**Issue by Phqzing** [Muqsit/InvMenu#188]
So when I remove the InvMenu window from the player it won't let me open any other inventory. Yes im using InvMenuCrashFix

PocketMine Version: 4.3.4
InvMenu Version: 4.3.11

code:
```
$menu->setListener(function(InvMenuTransaction $transaction):InvMenuTransactionResult
        {
            $player = $transaction->getPlayer();
            $clicked = $transaction->getItemClicked();

            foreach($this->plugin->getConfig()->get("gamemode-items") as $key => $arr)
            {
                if(!is_null($clicked->getNamedTag()->getTag("gitem")) and $clicked->getNamedTag()->getString("gitem") == $key)
                {
                    $world = $this->plugin->getServer()->getWorldManager()->getWorldByName($arr["arena"]);
                    if(!($world instanceof World)) return $transaction->discard();
                    $player->removeCurrentWindow();
                    return $transaction->discard()->then(function(Player $player)use($world, $arr):void
                    {
                        $this->plugin->giveKit($player, $arr["kit"]);
                        $player->teleport($world->getSpawnLocation());
                    });
                }
            }
            return $transaction->discard();
        });
        $menu->send($player);
```

**Comment by RazzyGP** [comment]
The same happens to me, due to an update in the pocketmine 4.3.4 InventoryTransactionEvent file

**Comment by dktapps** [comment]
InvCrashFix is no longer necessary in 4.3.4. It may cause new problems instead.

**Comment by Phqzing** [comment]
> InvCrashFix is no longer necessary in 4.3.4. It may cause new problems instead.

I tried it without the InvCrashFix and the same thing still happens

**Comment by Muqsit** [comment]
Fixed since [PocketMine v4.4.1](https://github.com/pmmp/PocketMine-MP/commit/e040c2b281a95269a69519ac88c7003277a427f0) and [InvMenu v4.3.12](https://github.com/Muqsit/InvMenu/commit/d93b1d96c0886abc0258de742149a53aed366db4)
